domain/src/main/java/org/oppia/android/domain/oppialogger/BUILD.bazel

```diff
@@ -0,0 +1,40 @@
+"""
+Library for providing logging to the Oppia android app.
+"""
+
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "oppia_logger",
+    srcs = [
+        "OppiaLogger.kt",
+    ],
+    visibility = ["//domain:__subpackages__"],
+    deps = [
+        "//domain/src/main/java/org/oppia/android/domain/oppialogger/analytics:controller",
+        "//model:event_logger_java_proto_lite",
+        "//utility/src/main/java/org/oppia/android/util/logging:console_logger",
+    ],
+)
+
+kt_android_library(
+    name = "startup_listener",
```

Comment:
This is actually a bit odd to be in this package. Could you maybe move ApplicationStartupListener to be in a new utility/startup package? It will be a small refactor, but I think it's a better place for the startup listener definition to live.

Ditto for ApplicationStartupListenerModule--that ought to be in utility/startup as well.

Reply:
Hi @BenHenning, you mean move the ApplicationStartupListenerModule from app into utility/startup